### PR TITLE
Add Sonar scans and code coverage

### PR DESCRIPTION
**Pull request by 1pretz1**
Co-auth: @batesyuk 

JIRA: https://govukverify.atlassian.net/jira/software/projects/GOVAPP/boards/405?selectedIssue=GOVAPP-441

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sonar.projectKey=alphagov_govuk-mobile-ios-app
+sonar.projectName=govuk-mobile-ios-app
+sonar.pullrequest.github.repository=alphagov/govuk-mobile-ios-app
+sonar.projectVersion=1.0
+
+sonar.organization=alphagov
+
+sonar.sources=Production/govuk_ios
```
Comment (batesyuk) [inline sonar-project.properties:8]: Just Production here I think

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+sonar.tests=Tests/govuk_ios
```
Comment (batesyuk) [inline sonar-project.properties:9]: Just Tests here

File: Fastlane/Fastfile [hunk 3]
```diff
@@ -36,6 +40,26 @@ platform :ios do
     )
   end
 
+  private_lane :code_coverage do
+    slather(
+      build_directory: configuration.app_output_directory,
+      output_directory: configuration.app_output_directory,
+      scheme: configuration.app_scheme,
+      sonarqube_xml: true,
+      proj: 'GovUK.xcodeproj'
```
Comment (batesyuk) [inline Fastlane/Fastfile:49]: Might be worth adding this to the build.yml?

File: sonar-project.properties [hunk 4]
```diff
@@ -0,0 +1,15 @@
+sonar.projectKey=alphagov_govuk-mobile-ios-app
+sonar.projectName=govuk-mobile-ios-app
+sonar.pullrequest.github.repository=alphagov/govuk-mobile-ios-app
+sonar.projectVersion=0.1
```
Comment (batesyuk) [inline sonar-project.properties:4]: I think we might need to update the version in the project to match this... something we need to spec so its always matching, I think

**Review by batesyuk — CHANGES_REQUESTED** [review]

File: sonar-project.properties (hunk 4 above)
Comment (1pretz1) [reply sonar-project.properties:4]: Done!

**Review by gclssvglx — APPROVED** [review]
Can only really review the SONAR scanner parts of this PR - that all looks great to me.

Great job @1pretz1 @batesyuk 👍

**Pull request merged**